mgmt/rest/rest_v1_test.go

```diff
@@ -858,9 +858,10 @@ func TestV1Task(t *testing.T) {
 			// GetTasks returns an unordered map,
 			// thus there is more than one possible response
 			So(
+				string(body),
+				ShouldBeIn,
 				responses,
-				ShouldContain,
-				string(body))
+			)
```

Comment:
In this test file there are only changes related to keeping the pattern that an actual result should be at the first position:
```
So(actual-result, assertion, expected-result)
```